robot/basestation/static/js/roslibjs/ros-helpers.js

```diff
@@ -2,6 +2,15 @@ REQUEST_TIMEOUT = 3000
 ROTATE_TIMEOUT = 1000
 lastRotate = 0
 
+// minimum and maximum acceptable battery voltages (volts)
+MIN_VOLTAGE = 12.5
+MAX_VOLTAGE = 16.9
```

Comment:
Please change this to 16.8 and make sure that the error modal only appears when it is strictly _greater_ than 16.8 (so 16.8 is allowed). The rest are fine, it's just this one that needs to be changed in this sense.